src/site/content/en/feed.njk

```diff
@@ -29,7 +29,8 @@ permalink: "/{{lang}}/feed.xml"
   <icon>{{ metadata.icon }}</icon>
   <logo>{{ metadata.logo }}</logo>
   {% for post in collections.blogPosts -%}
-  {% set absolutePostUrl -%}{{ post.url | stripLanguage | url | absoluteUrl(metadata.url) }}{%- endset %}
+  {% set postUrl = '/' + post.fileSlug + '/' %}
+  {% set absolutePostUrl -%}{{ postUrl | url | absoluteUrl(metadata.url) }}{%- endset %}
```

Comment:
it's not important but why doesn't this one reference `metadata`?

Reply:
I think it may come from this: https://www.11ty.dev/docs/plugins/rss/#supplies-the-following-nunjucks-filters

nunjucks syntax is super gross, but I think to pass two arguments to a filter you have to call it as:

```
foo | bar(baz) // == bar(baz, foo)
```

Reply:
@devnook see note above about adding a comment (maybe at the top of this file?) to https://www.11ty.dev/docs/plugins/rss/#supplies-the-following-nunjucks-filters so folks know where these filters come from